code/ac/llama/Token.hpp

```diff
@@ -3,8 +3,17 @@
 //
 #pragma once
 #include <cstdint>
+#include <vector>
 
 namespace ac::llama {
 using Token = std::int32_t;
 inline constexpr Token Token_Invalid = -1;
+
+struct TokenData {
```

Comment:
I've added this here since I needed it in the `Sampler` and `Session`. I'm not sure if here is the best place to stay, so I'm open for ideas how to move it.